ActionGame/ActionGame/Script/Header/sc_Result.h

```diff
@@ -0,0 +1,25 @@
+
+#ifndef SC_RESULT_H
+#define SC_RESULT_H
+
+class Scene_Result : public SceneBase {
```

Comment:
クラス名にアンダースコアはあまり使わないです。
基底クラスは `XxxBase` とすることが多いので「Sceneクラス(群)の基底クラス」は `SceneBase` で良いと思います。
Xxxの部分は「Scene」のように名詞ですね。

派生クラスは `YyyXxx` とする方が良いです。
Xxxの部分は基底クラスに出てきた「Scene」等で、それをYyyで修飾する形です。

そのような命名にして英語として違和感が無い名前にしましょう。
今回の場合だと
```suggestion
class ResultScene : public SceneBase {
```
で問題無いと思います。

Reply:
@taishi-matsumura 
d3192e7 で修正しました。

`OnPlayScene`だとなんか気持ち悪かったので`GameScene`に名前を変えました。